package.json

```diff
@@ -58,6 +58,7 @@
     "eslint-plugin-vue": "^7.20.0",
     "execa": "^5.1.1",
     "flow-bin": "0.139.0",
+    "gitpkg": "^1.0.0-beta.2",
```

Comment:
Is this used?

Reply:
Was using gitpkg to publish to a git ref in order to install in another codebase test the plugin there. Will clean it up !